orga/translations/en.json

```
@@ -75,6 +75,10 @@
       },
       "archived": "Archived campaign",
       "code": "Code",
+      "credits": {
+        "number":"{count, plural, =0 {0 credit} =1 {1 credit} other {{count} credits}}",
```

En anglais, le 0 est pluriel.
```suggestion
        "number":"{count, plural, =0 {0 credits} =1 {1 credit} other {{count} credits}}",
```